### PR TITLE
#1314: Introduce new org.eclipse.persistence.jpars.server artifact

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

**Review by rfelcman — APPROVED** [review]
LGTM except mentioned minor issues above

File: jpa/org.eclipse.persistence.jpars.server/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates. All rights reserved.
```
[the remark this answers is not in the document]
Comment (lukasj) [reply jpa/org.eclipse.persistence.jpars.server/pom.xml]: done

File: jpa/org.eclipse.persistence.jpars.server/pom.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+
+-->
+
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>EclipseLink EclipseLink JPA-RS Server</name>
```
[the remark this answers is not in the document]
Comment (lukasj) [reply jpa/org.eclipse.persistence.jpars.server/pom.xml]: done

File: pom.xml [hunk 3]
```diff
@@ -256,6 +256,7 @@
         <module>jpa/org.eclipse.persistence.jpa</module>
         <module>jpa/org.eclipse.persistence.jpa.modelgen</module>
         <module>jpa/org.eclipse.persistence.jpars</module>
+        <module>jpa/org.eclipse.persistence.jpars.server</module>
```
[the remark this answers is not in the document]
Comment (lukasj) [reply pom.xml:259]: I prefer alphabetical ordering. Maven knows the correct order...

**Pull request merged**